src/ia-menu-slider.js

```diff
@@ -59,13 +65,50 @@ export class IAMenuSlider extends LitElement {
     return this.selectedMenu ? 'open' : 'closed';
   }
 
+  /**
+   * closes menu drawer
+   */
+  closeMenu() {
+    this.open = false;
+    const drawerClosed = new CustomEvent('ItemNavMenuClosed', {
```

Comment:
QUESTION: I sometimes think about the risks of using free strings here... so easy to misspell. Should we be using constants instead? (Allow for static checking too!)

Reply:
I've moved it up to its own object so we can use a reference in the code.
I'll think about it further and consult our lit element team to see what is the best approach on an ecosystem level as there are other places that this event name is being referenced.